WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormTableViewModel.swift

```diff
@@ -62,6 +66,17 @@ enum ProductFormSection: Equatable {
                 self.isSwitchOn = isSwitchOn
             }
         }
+
+        /// View model for warning UI
+        struct WarningViewModel: Equatable {
+            let icon: UIImage
+            let title: String?
+
+            init(icon: UIImage, title: String?) {
+                self.icon = icon
+                self.title = title
+            }
```

Comment:
This does not need to be defined since it's auto-generated. 🙂 

```suggestion

```

Reply:
updated in: https://github.com/woocommerce/woocommerce-ios/pull/2627/commits/21d09a9a1a1164dbe22467c6a0ce15b74610f3fa